src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -65,6 +65,8 @@
     // get realigned incorrectly.  See https://github.com/broadinstitute/gatk/issues/5060
     public static final int MINIMUM_READ_LENGTH_AFTER_TRIMMING = 10;
 
+    // this notation can be interpreted as a representation of the alleles present on the two phased haplotypes at the site:
```

Comment:
What about mixed hets with no ref allele or span del?  Is there any reason it can't be simply 0: first allele in the VariantContext's allele list, 1: second allele in same list?

Reply:
I've left the current functionality as is, in that it only phases sites that are biallelic with a single site-specific alternate allele. Even with this restriction, though, we can't guarantee the order of alternate alleles in the VC's allele list, and in some cases the * allele ends up at index 1 (ie the allele list is REF, *, ALT), in which case the indexing schemes become more confusing.